Airports/EGBK/SMR/Geo.txt

```diff
@@ -0,0 +1,147 @@
+EGBK Northampton Sywell               S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
+;A1
+N052.18.04.317 W000.47.41.323 N052.18.04.053 W000.47.41.033 smrYellow
+;D1
+N052.18.06.693 W000.47.47.604 N052.18.06.471 W000.47.48.412 smrYellow
+;Taxiline
```

Comment:
If you can, name this with the actual taxiway name.  If it was taxiway A you might do:
```suggestion
;Taxiline_A
```
Same with the other ones as well please!